tests/stats_monitor/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+add_test(NAME stats_monitor
+         WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+         CONFIGURATIONS hindsight
+         COMMAND sh -c "rm -rf output; $<TARGET_FILE:hindsight_cli> hindsight.cfg 7")
```

Comment:
Integration tests are generally run against the installed packages (they are our package sanity checks too).  This should be calling run.sh